index.html

```diff
@@ -104,6 +105,45 @@ <h5>Streaming vs. Saving</h5>
         <p class="link"></p>
       </div>
       <!-- End Streaming vs. Saving -->
+      
+      <!-- Troubleshooting Common Issues -->
+      <label class="recipe" for="troubleshooting">Trouble Shooting Common Issues</label>
+       <input type="checkbox" id="troubleshooting">
+      <div class="hiding"> 
+          <h5>Troubleshooting Common Issues</h5>
+          <p>There are many common issues you may encounter when using FFmpeg. This section is here to provide some possible explanations and solutions.For more general solutions to issues regarding download, installation, and other possible questions or issues, please refer to the <a href="https://ffmpeg.org/faq.html">FFmpeg FAQ.</a>
+<p>Before delving into some specific error messages and what may cause them, there are a few general tips you can follow to help make sure your files are doing what you want them to.
```

Comment:
Not sure if you wanted to have these three paragraphs split -- I think it looks fine either way, but when it's rendered as code, it'll have all three of these (line 115 to 117) as one paragraph. You'd have to wrap the others in `<p>` tags if you want them split up. Again, no preference here, it's up to you.